lib/matplotlib/testing/__init__.py

```diff
@@ -105,6 +105,12 @@ def subprocess_run_for_testing(command, env=None, timeout=60, stdout=None,
             import pytest
             pytest.xfail("Fork failure")
         raise
+    except subprocess.CalledProcessError as e:
```

Comment:
I’ve not checked the details, but seems to be similar to  #29890. Please check and decide what’s the better approach.

Reply:
The logging version plays nicer with pytest's tooling, will drop that commit.